pkg/operation/common/utils.go

```diff
@@ -318,56 +317,47 @@ func DeleteVpa(ctx context.Context, c client.Client, namespace string, isShoot b
 	return nil
 }
 
-// DeleteLoggingStack deletes all resource of the EFK logging stack in the given namespace.
-func DeleteLoggingStack(ctx context.Context, k8sClient client.Client, namespace string) error {
-	if k8sClient == nil {
-		return errors.New("must provide non-nil kubernetes client to common.DeleteLoggingStack")
-	}
-
-	// Delete the resources below that match "gardener.cloud/role=logging"
-	lists := []client.ObjectList{
-		&corev1.ConfigMapList{},
-		&batchv1beta1.CronJobList{},
-		&rbacv1.ClusterRoleList{},
-		&rbacv1.ClusterRoleBindingList{},
-		&rbacv1.RoleList{},
-		&rbacv1.RoleBindingList{},
-		&appsv1.DaemonSetList{},
-		&appsv1.DeploymentList{},
-		// TODO: Use autoscaling/v2beta2 for Kubernetes 1.19+ shoots once kubernetes-v1.19 golang dependencies were vendored.
-		&autoscalingv2beta1.HorizontalPodAutoscalerList{},
-		&extensionsv1beta1.IngressList{},
-		&corev1.SecretList{},
-		&corev1.ServiceAccountList{},
-		&corev1.ServiceList{},
-		&appsv1.StatefulSetList{},
-	}
-
-	for _, list := range lists {
-		if err := k8sClient.List(ctx, list,
-			client.InNamespace(namespace),
-			client.MatchingLabels{v1beta1constants.GardenRole: v1beta1constants.GardenRoleLogging}); err != nil {
-			return err
-		}
+// DeleteShootLoggingStack deletes all shoot resource of the logging stack in the given namespace.
+func DeleteShootLoggingStack(ctx context.Context, k8sClient client.Client, namespace string) error {
```

Comment:
Some naming suggestions as it:
- deletes only loki related resources
- is used to delete them also in the `garden` namespace

```suggestion
// DeleteLoki  deletes all resources of the Loki in a given namespace.
func DeleteLoki(ctx context.Context, k8sClient client.Client, namespace string) error {
```